nifi-docs/src/main/asciidoc/expression-language-guide.adoc

```diff
@@ -963,6 +963,36 @@ Expressions will provide the following results:
 	will return 0.
 
 
+[.function]
```

Comment:
Sounds good to me, will update the description of the Boolean Logic section to read "Each of these functions are designed to work on values of type Boolean"